README.md

```diff
@@ -1 +1,37 @@
-# ccs-scale-infra-db-bat
\ No newline at end of file
+# ccs-scale-infra-db-bat
+
+## SCALE Buy a Thing (BaT) databases
+
+### Overview
+This repository contains a complete set of configuration files and code to deploy SCALE shared service databases into the AWS cloud.  The infrastructure code is written in [Terraform](https://www.terraform.io/).
+
+
+### Prerequisites
+
+Ensure that the following SSM parameters have been configured in AWS:
+
+```
+/bat/{env}-spree-db-master-username
+/bat/{env}-spree-db-master-password
+
+/bat/{env}-spree-db-app-username
+/bat/{env}-spree-db-app-username
+```
+
+### Post Create
+
+After the database has been provisioned, we need to create a user for the spree application to use to connect.
+
+```
+CREATE USER {USERNAME} WITH ENCRYPTED PASSWORD '{PASSWORD}';
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO {USERNAME};
+GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA public TO {USERNAME};
+GRANT ALL PRIVILEGES ON DATABASE spree TO {USERNAME};
```

Comment:
Was wondering if we could do this with a local-exec provisioner - probably, but would involve installing and configuring psql client tooling as part of the CodeBuild install phase - and picking out the SSM params to use in the command - not worth it now but something we might want to look at for future DB provisioning and/or running our version update scripts as part of a DB pipeline??

Reply:
Yep - be good to automate the db scripts (certainly the version update scripts for prod releases) at some point if we get the time.

This wasn't the most elegant solution, but pragmatic for now given the time constraints. Maybe if it turns out to be troublesome we can revisit it.